src/toastfile.rs

```diff
@@ -12,6 +13,67 @@ pub const DEFAULT_LOCATION: &str = "/scratch";
 // The default user for commands and files copied into the container
 pub const DEFAULT_USER: &str = "root";
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct MappingPath {
+    pub host_path: PathBuf,
+    pub container_path: PathBuf,
+}
+
+impl Serialize for MappingPath {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serializer.serialize_str(
+            format!(
+                "{}:{}",
+                self.host_path.to_str().unwrap(),
+                self.container_path.to_str().unwrap(),
```

Comment:
I see we use `to_string_lossy` when using these fields in `docker.rs`. Can we use that here instead of the dangerous `unwrap` calls?